src/Mock/MockGenerator.php

```diff
@@ -469,6 +473,13 @@ private function generateMethods(
                     "(__FUNCTION__)->invokeWith(\n" .
                     '            new \Eloquent\Phony\Call\Arguments' .
                     "(\$arguments)\n        );";
+            } else if ($isNeverReturn) {
+                $body .=
+                    "\n        }\n\n" .
+                    "        ${handle}->spy" .
```

Comment:
```suggestion
                    "        {$handle}->spy" .
```

`${var}` is deprecated in PHP 8.2, see #257